consistency-checker/src/main/java/com/redhat/lightblue/migrator/consistency/ConsistencyChecker.java

```diff
@@ -202,7 +202,9 @@ public void run() {
             findRequest.where(and(
                     withValue("configurationName = " + configuration.getConfigurationName()),
                     withValue("whenAvailableDate <= " + ClientConstants.getDateFormat().format(new Date())),
-                    not(withSubfield("jobExecutions", withValue("completedFlag = true")))));
+                    not(withSubfield("jobExecutions", withValue("jobStatus = 'COMPLETED_SUCCESS'"))),
+                    not(withSubfield("jobExecutions", withValue("jobStatus = 'COMPLETED_PARTIAL'")))
```

Comment:
OK. I will update the query in this PR. I will also introduce the changes to change the status we previously discussed